.eslintrc.yml

```diff
@@ -0,0 +1,139 @@
+parserOptions:
```

Comment:
And even if there wasn't one, it would be a matter of a different commit and a different PR.